distopia/pydistopia.pyx

```diff
@@ -874,6 +882,10 @@ def calc_self_distance_array_triclinic(
     dims[0] = <ssize_t > final_size
 
 
+    # return early, will seg
+    if nvals0 == 0:
```

Comment:
very nitpicky but this probably should be typed either to f32/64 based on the input